driveai.org/docs/index.md

```diff
@@ -6,8 +6,6 @@ DriveAI is a 501(c)(3) nonprofit organization dedicated to the research and deve
 
 Our goal as an organization is to hold regular meetings to discuss developments in vehicle autonomy, create resources for the general public to learn about vehicle autonomy, and build guides to help hobbyists and entrepreneurs development with autonomous vehicle development. We'll know we accomplished this when anybody can become comfortable and up-to-date enough with autonoumous vehicles to build their own.
 
-If you share our vision and want to help, check out our [Get Involved page](http://media2.giphy.com/media/fLgb07Tan59n2/giphy.gif).
```

Comment:
Lol, this still needs to exist, we can't just get rid of it, just use the correct link